rest.http

```diff
@@ -31,8 +46,9 @@ Content-Type: application/json
 }
 
 # Save bookmark (no timestamp)
-# This will use current time
+# This will use current UTC time
```

Comment:
Time zone does not matter. Current time on Earth does not depend on where you are.